simpleflu/README.md

```diff
@@ -33,14 +28,11 @@ simulation {
 }
 ```
 
-The only additional content in this file is the line `include simpleflu.fred`, which imports the `INFLUENZA` condition and associated states from `simpleflu.fred`.
```

Comment:
I want to try and stay away from the term "import" here since this has a different meaning in the FRED language.